vars.tf

```diff
@@ -102,3 +102,14 @@ variable "subdomain" {
 variable "tf_remote_common" {
   description = "Path to the Common remote, in `org/workspace` syntax."
 }
+
+variable "customer" {
+  description = "Customer name, used in AWS tags"
+  type        = string
+}
+
+variable "create_dashboard" {
+  description = "Set to 1 to create a CloudWatch dashboard"
```

Comment:
Instead of `1`, should this be `true`?

Reply:
Yep, I changed my mind and forgot to update the description. Thanks.